### PR TITLE
[ru] Localize `Contributor ladder`

### LINKED ISSUE #3052: [ru] Localize `contributor-ladder`
### Description

This Issue is to localize [`contributor-ladder`](https://github.com/cncf/glossary/blob/main/content/en/contributor-ladder/_index.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by kirkonru**
### Describe your changes
This PR adds translation for `Contributor ladder`

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/contributor-ladder/_index.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+## 2. Апруверы
+
+Апруверы дают обратную связь по PR'ам и одобряют их. Любой активный участник может стать апрувером (см. [Стать апрувером](#стать-апрувером)). 
+В Глоссарии различают два вида апруверов: (1) апруверы для оригинальной английской версии глоссария и (2) апруверы для команд локализации.
+
+Апруверы глоссария должны:
+
+- Проверять PR на техническую точность; 
+- Закреплять Issues за авторами и навешивать на них соответствующие лейблы;
+- Давать контрибьюторам обратную связь и направлять их при необходимости,
```
Comment (shurup) [inline content/ru/contributor-ladder/_index.md:40]:
```suggestion
- Давать контрибьюторам обратную связь и направлять их при необходимости;
```

File: content/ru/contributor-ladder/_index.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+- Вычитывать и редактировать материалы.
+
+Если апрувер более не заинтересован в выполнении вышеперечисленных обязанностей или не может их выполнять, он должен сообщить об этом мейнтейнерам и сложить свои полномочия.
+
+### Апруверы английской версии Глоссария
+
+Различают три вида апруверов:
+
+1) Апруверы с богатым техническим бэкграундом.
```
Comment (shurup) [inline content/ru/contributor-ladder/_index.md:49]:
```suggestion
1) Апруверы с богатым техническим опытом.
```

File: content/ru/contributor-ladder/_index.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+Однако, проверяя PR, они не должны ограничиваться его технической стороной — следует убедиться, что его проверит редактор.
+
+**Редакторы**: Редакторы вычитывают термины и следят за тем, чтобы изложение велось простым языком в соответствии с Руководством по стилю. 
+Если в описание термина вносились серьезные изменения, редактор должен попросить технического специалиста просмотреть его еще раз, чтобы убедиться, что смысл сохранился.
+
+### Апруверы локализаций
+
+У глоссария также есть апруверы локализаций, которые принадлежат к одной из команд локализации (команды, занимающейся переводом глоссария). 
+Апруверы локализации занимаются проверкой локализованных терминов и могут мержить PR'ы в соответствующую ветку локализации. 
+Любой апрувер локализации также может апрувером для английской версии глоссария, если он соответствует требованиям.
```
Comment (shurup) [inline content/ru/contributor-ladder/_index.md:65]:
```suggestion
Любой апрувер локализации также может быть апрувером для английской версии глоссария, если соответствует требованиям. 
```

File: content/ru/contributor-ladder/_index.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+Апруверы локализации занимаются проверкой локализованных терминов и могут мержить PR'ы в соответствующую ветку локализации. 
+Любой апрувер локализации также может апрувером для английской версии глоссария, если он соответствует требованиям. 
+
+### Стать апрувером
+
+Кандидаты в апруверы должны иметь опыт подачи качественных PR'ов и помощи другим в приведении их PR'ов в состояние, пригодное для слияния. 
+Если позволяет часовой пояс, они также должны регулярно посещать встречи [Рабочей группы по глоссарию](https://www.cncf.io/calendar/).
+
+Чтобы стать апрувером, прежде всего расскажите о своем желании мейнтейнерам.
+Они попросят вас продемонстрировать вышеуказанные навыки, внося PR'ы, делая обзоры и выполняя другие подобные задачи под их руководством.
```
Comment (shurup) [inline content/ru/contributor-ladder/_index.md:73]:
```suggestion
Они попросят вас продемонстрировать вышеуказанные навыки, внося PR'ы, делая рецензии и выполняя другие подобные задачи под их руководством. 
```

File: content/ru/contributor-ladder/_index.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+Если мейнтейнер более не заинтересован в выполнении перечисленных выше обязанностей или не может их выполнять, ему следует перевести себя в эмерит-статус (почетного участника).
+
+### Стать мейнтейнером
+
+Мейнтейнеры должны проявить себя как успешные апруверы, а также как авторы ряда качественных PR'ов. 
+Если позволяет часовой пояс, они должны регулярно посещать встречи рабочей группы по глоссарию.
+
+Прежде всего расскажите существующим мейнтейнерам о своем желании присоединиться к ним.
+Существующие мейнтейнеры попросят вас продемонстрировать вышеупомянутую квалификацию, внося PR, делая обзоры и выполняя другие подобные задачи под их руководством.
```
Comment (shurup) [inline content/ru/contributor-ladder/_index.md:95]:
```suggestion
Существующие мейнтейнеры попросят вас продемонстрировать вышеупомянутую квалификацию, внося PR, делая рецензии и выполняя другие подобные задачи под их руководством. 
```

File: content/ru/contributor-ladder/_index.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+### Стать апрувером
+
+Кандидаты в апруверы должны иметь опыт подачи качественных PR'ов и помощи другим в приведении их PR'ов в состояние, пригодное для слияния. 
+Если позволяет часовой пояс, они также должны регулярно посещать встречи [Рабочей группы по глоссарию](https://www.cncf.io/calendar/).
+
+Чтобы стать апрувером, прежде всего расскажите о своем желании мейнтейнерам.
+Они попросят вас продемонстрировать вышеуказанные навыки, внося PR'ы, делая обзоры и выполняя другие подобные задачи под их руководством. 
+Через некоторое время мейнтейнеры решат, давать ли вам статус апрувера. 
+Решение будет основано на показанном вами уровне мастерства и ответственности.
```
Comment (shurup) [inline content/ru/contributor-ladder/_index.md:75]: Personally, I like the idea of _responsibility_ more, but let's stick with the original wording (i.e. _responsiveness_).
```suggestion
Решение будет основано на показанном вами уровне мастерства и отзывчивости.
```

File: content/ru/contributor-ladder/_index.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+### Стать мейнтейнером
+
+Мейнтейнеры должны проявить себя как успешные апруверы, а также как авторы ряда качественных PR'ов. 
+Если позволяет часовой пояс, они должны регулярно посещать встречи рабочей группы по глоссарию.
+
+Прежде всего расскажите существующим мейнтейнерам о своем желании присоединиться к ним.
+Существующие мейнтейнеры попросят вас продемонстрировать вышеупомянутую квалификацию, внося PR, делая обзоры и выполняя другие подобные задачи под их руководством. 
+По итогам совместной работы они решат, готовы ли вы стать мейнтейнером. 
+Их решение будет основано на проявленном уровне мастерства и ответственности.
```
Comment (shurup) [inline content/ru/contributor-ladder/_index.md:97]:
```suggestion
Их решение будет основано на проявленном уровне мастерства и отзывчивости.
```

File: content/ru/contributor-ladder/_index.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+## 4. Менеджеры сообщества
+
+Менеджеры сообщества создают гостеприимное и увлекательное сообщество. Любой участник сообщества может стать его менеджером. От них требуется:
```
Comment (shurup) [inline content/ru/contributor-ladder/_index.md:101]:
```suggestion
Менеджеры сообщества создают гостеприимное и вовлеченное сообщество. Любой участник сообщества может стать его менеджером. От них требуется:
```

**Review by shurup — COMMENTED** [review]
In general, LGTM! I have just a few minor corrections.

**Comment by kirkonru** [comment]
> In general, LGTM! I have just a few minor corrections.

Thanks a lot! 🤗

**Review by shurup — APPROVED** [review]
Thanks for these corrections, it's 100% LGTM now! :sweat_smile:

**Review by tym83 — APPROVED** [review]

**Pull request merged**